### PR TITLE
test qkcpow case

### PR DESCRIPTION
**Pull request by tomdong2010**

### CODE REVIEW

File: consensus/qkchash/qkchash_test.go [hunk 1]
```diff
@@ -126,3 +126,29 @@ func sealBlock(t *testing.T, q *QKCHash, h *types.RootBlockHeader) {
...
+	err = q.VerifySeal(nil, header, big.NewInt(10))
+	assert.NoError(err, "should have correct nonce / mix digest")
+
+	// Wrong
+	header.Nonce = block.IHeader().GetNonce() - 1
+	err = q.VerifySeal(nil, header, big.NewInt(10))
+	assert.Error(err, "should have error because of the wrong nonce")
+	//wrong epoch
+	header.Nonce = block.IHeader().GetNonce()
+	err = q.VerifySeal(nil, header, big.NewInt(1000))
```
Comment (syntrust) [inline consensus/qkchash/qkchash_test.go]: why change diff?

Comment (tomdong2010) [reply consensus/qkchash/qkchash_test.go]: test VerifySeal  to show  err        maybe add instroduce about it

Comment (syntrust) [reply consensus/qkchash/qkchash_test.go]: it's about  epoch, so change height as python does

Comment (syntrust) [reply consensus/qkchash/qkchash_test.go]: impossible and meaningless to change height without change header. remove the epoch case.

**Pull request merged**